CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Unreleased
-- [#844](https://github.com/kubernetes-sigs/federation-v2/pull/844)
+- [#844](https://github.com/kubernetes-sigs/federation-v2/pull/844) -
    `kubefedctl federate` namespace with its content gets an option to
    skip API Resources via `--skip-api-resources`.
-
+-  [#832](https://github.com/kubernetes-sigs/federation-v2/issues/832) -
```

Comment:
(No action required) I've been inserting new entries at the top, but I guess it doesn't really matter.

Reply:
Updated! Better to follow a common protocol.